lib/Importer/Caffe2ModelLoader.cpp

```diff
@@ -579,28 +579,35 @@ void Caffe2ModelLoader::loadWeight(const caffe2::OperatorDef &op) {
     auto dim = getShape(dict["shape"]);
 
     size_t i = 0;
+#define LOAD_TENSOR_FILL(TYPE_NAME, NATIVE_TYPE, PROTO_TYPE_NAME)              \
+  T->reset(ElemKind::TYPE_NAME, dim);                                          \
+  auto TH = T->getHandle<NATIVE_TYPE>();                                       \
+  for (auto num : dict["values"]->PROTO_TYPE_NAME()) {                         \
```

Comment:
If a `protobuf::RepeatedField` is convertible to `llvm::ArrayRef` this could be `TH = dict["values"]->PROTO_TYPE_NAME();` which looks cool.  But it's probably not :)

Reply:
Yup, not convertible :/